### DISCUSSION TITLE
camera projection matrix

### DISCUSSION THREAD

**Discussion by Zzerr0r** [microsoft/AirSim#4506] [Support Q&A]
The API provides a camera projection matrix through client.simGetCameraInfo().proj_mat however I don't know if this is the intrinsic or extrinsic matrix. 
I am trying to reconstruct the world position of an object by multiplying its representation on the image plane with the inverse of the projection matrix. 
does anyone have experience with this?

**Comment by sandilyasg** [comment]
@Zzerr0r any update on this? I am trying to use simGetCameraInfo with Unity API but it is crashing

**Reply by Zzerr0r** [reply]
@sandilyasg I have no experience with Unity sorry
